proposals/models.py

```diff
@@ -703,13 +703,7 @@ def generate_pdf(self, force_overwrite=False):
 
     @property
     def pdf_template_name(self):
-        """Determine the correct PDf template for this proposal."""
-        template_name = 'proposals/proposal_pdf.html'
-        if self.is_pre_approved:
-            template_name = 'proposals/proposal_pdf_pre_approved.html'
-        elif self.is_pre_assessment:
-            template_name = 'proposals/proposal_pdf_pre_assessment.html'
-        return template_name
+        return 'proposals/proposal_pdf.html'
```

Comment:
I agree that this is redundant now; differences should be handled by the new PDF framework

Reply:
I got rid of this now, and its reference in the `ProposalAsPDF` view. This works fine for me. I was wondering does this now require a migration?

Reply:
No it doesn't. Migrations are only needed for ORM Fields (which are always class variables, and (almost) always named `xField`). This is a class method, those _should_ never trigger migrations.

If you are unsure, you can run `python manage.py makemigrations --dry-run`. This will only display the migrations Django _wants_ to create, without actually creating them. 

Alternatively, we now have a check for PR's that will tell you if you are missing a migration. The latest run passed successfully: https://github.com/DH-IT-Portal-Development/ethics/actions/runs/6847879373/job/18616976001?pr=574